libretro_core_options.h

```diff
@@ -476,6 +476,16 @@ struct retro_core_option_definition option_defs_us[] = {
       },
       "disabled"
    },
+   {
+      BEETLE_OPT(pal_video_override),
+      "PAL (European) video timing override",
+      "Due to different standards PAL games often appear slowed down compared to the American or Japanese NTSC releases. This option can be used to override the timings in order to attempt to run these games with the NTSC framerate.",
+      {
+         { "no_override", "Standard PAL timings" },
+         { "fast_pal",  "PAL resolution, NTSC framerate" },
+      },
+      "no_override"
```

Comment:
I see that there's now a new file for internationalization, should it be updated as well or will it fail over to the English version for options that haven't been translated? Unfortunately I don't speak any of the currently implemented languages well enough to provide the translation...

Reply:
The way it was explained to me in pull request https://github.com/libretro/beetle-psx-libretro/pull/608 is that internationalized core options work as overrides, so it is safe for us not to update the translations when adding new core options.